package.json

```diff
@@ -160,9 +160,9 @@
         "idb-keyval": "^6.2.1",
         "lodash": "^4.17.21",
         "luxon": "^3.4.4",
-        "react": "^16.14.0",
-        "react-dom": "^16.14.0",
-        "react-helmet": "^6.1.0",
+        "react": "^18.3.1",
+        "react-dom": "^18.3.1",
+        "react-helmet-async": "^2.0.5",
```

Comment:
Added react-helmet-async as old react-helmet was not supporting react18, test cases were failing